frontend/src/app/components/Visibility/tests.js

```diff
@@ -0,0 +1,78 @@
+/* global describe, beforeEach, afterEach, it */
```

Comment:
Instead of specifying the methods individually, we could just set the `mocha` ESLint environment: `/* eslint-env mocha */`

Or, possibly look into using ESLint config `overrides` with a glob to automatically set the `mocha` env for `frontend/src/**/tests.js` files:
https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns

Reply:
Yeah, that's a good idea....could you file an issue to track?

Reply:
b/c we do this in all the places